CONTRIBUTING.md

```diff
@@ -1,16 +1,13 @@
 # Contributing
 
-Contributions are always welcome. Before contributing please read the
-[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](issues); your issue may have already been discussed or fixed. To contribute,
```

Comment:
was this bit about code of conduct removed on purpose?

Reply:
Yes.  My understanding is that we are not supposed to have individual CoC files in our repos; in fact, I think, the repolinter will complain if one is present.

Reply:
maybe we should be consistent with Agent repo, which links to the company-wide one: https://github.com/newrelic/newrelic-dotnet-agent/blob/main/CONTRIBUTING.md

?

Reply:
adding for context from Slack convo:
" several other team's repos and their README and CONTRIBUTING files and none of them had a CoC link in them."

Reply:
I took your suggestion and re-added a link to the company-wide CoC in the CONTRIBUTING.md file.